ui/templates/welcome.html

```diff
@@ -6,9 +6,26 @@
 
 <p>
```

Comment:
All we want here are the links to the respositories. The user will just click through to the listing page and then import from there. L21-27 could just go.